runtime/pkg/graceful/http.go

```diff
@@ -11,26 +11,37 @@ import (
 
 const httpShutdownTimeout = 15 * time.Second
 
+type Options struct {
+	Port     int
+	CertPath string
+	KeyPath  string
```

Comment:
But it's fine to keep these names since hopefully it's clear from the surrounding context that it's TLS related